tests/main/prepare-image-validation-sets/task.yaml

```diff
@@ -0,0 +1,162 @@
+summary: Verify that we are able to use validation sets with the model assertion through prepare-image 
+
+# autopkgtest run only a subset of tests that deals with the integration
+# with the distro
+backends: [-autopkgtest]
+
+# disable the following distributions
+# ubuntu-14, lack of systemd-run
+# ubuntu-18.04-32: pc gadget from 20 is not available
```

Comment:
Should this  perhaps be PC gadget for 18?
Should any of these become #TODOs ?

Reply:
Good questions! However it's okay, the test still runs for 18 (64-bits), it's because the gadget is 64 bits (core20 only exists as 64 bit) and not 32. It's not super important to run this spread test on all possible spread platforms.

Reply:
OK thanks